package.json

```diff
@@ -27,6 +27,8 @@
     "coverage": "scripts/coverage.sh"
   },
   "devDependencies": {
+    "@babel/core": "^7.5.5",
```

Comment:
For what reason do we pull this in again? 🙂

Reply:
Parcel did this. Probably a bug, will remove, thanks!